plugins/kubernetes/app/views/kubernetes_releases/_create_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<script type='text/ng-template' id='kubernetes/kubernetes_release_wizard.tmpl.html'>
+  <div class="dialog kubernetes-release-dialog" ng-controller="KubernetesReleaseWizardCtrl">
+    <header>
+      <h2 class="title">Create a new release</h2>
+      <span class="step">Step {{current}} of 2</span>
```

Comment:
True. :-D

I'm gonna change it. Honestly, it crossed my mind, but then I thought: we'll never gonna change the number of steps. :-D